src/velangtemp_data_provider.cpp

```diff
@@ -46,14 +46,6 @@ imu_ros2::msg::AdiImuData AdiImuDataProvider::getData(bool& success)
 
     if(success)
     {
```

Comment:
this is related to how the device is working: for burst_data_selection 1 you will have available delta angle and delta velocity data when reading from the buffer, and for burst_data_selection 0 you will have accelerometer and gyroscope data when reading from the buffer. this configuration has to be added in the code before retrieving the data.

Reply:
As discussed, this will be implemented in another PR. The current PR looks good to me.